scripts/bootstrap-prefix.sh

```diff
@@ -45,11 +45,20 @@ emake() {
 
 efetch() {
 	if [[ ! -e ${DISTDIR}/${1##*/} ]] ; then
-	  	if [[ ${OFFLINE_MODE} ]]; then
-		  echo "I need ${1##*/} from $1 in $DISTDIR, can you give it to me?"
-		  read
-		  [[ -e ${DISTDIR}/${1##*/} ]] && return 0
-		  # Give fetch a try
+		mkdir -p "${DISTDIR}" >& /dev/null
+
+		# Try fetching from local mirrors first, as this requires no connection
+		for loc in ${GENTOO_MIRRORS} ; do
```

Comment:
Is this behaviour common for GENTOO_MIRRORS?  Else I'd prefer to use a different var that denotes paths that can be looked up files in.  E.g. over NFS or a local store indeed.

Reply:
https://github.com/gentoo/portage/blob/fe8701c/lib/portage/package/ebuild/fetch.py#L957-L963

Reply:
Apparently this is not the behavior of `emerge-webrsync`, only `emerge` itself does this. Which is a shame because in tools that don't check /etc/portage/mirrors this is the only variable usable to copy files over from a read-only DISTDIR without everything breaking.

Reply:
FWIW, it's also documented in `man 5 make.conf`: 
![imatge](https://user-images.githubusercontent.com/1339276/209331637-f155e214-e48a-49bd-bc9b-6e87861a2d9d.png)

Reply:
I learned something new :)  This is useful.  Sounds like we're actually adding some good (pre-existing) feature with this.